sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1739,6 +1978,12 @@ wait list parameter, the semantics are that the barrier node being added to
 will depend on all the existing graph leaf nodes, not only the leaf nodes
 that were added from the queue being recorded.
```

Comment:
Is it really the case still that barriers cannot be used when creating a graph via the explicit API?  It seems weird that we allow them when they use `dynamic_event` but not when they take a regular `event`.